examples/sensor_metadata_schema/sensor_metadata_schema.json

```diff
@@ -0,0 +1,84 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Waste Package Monitoring Sensor Metadata",
+    "description": "Metadata for a waste package monitoring sensor, including its unique identifier, position, and measurement capabilities.",
+    "type": "object",
+    "required": [
+        "sensor_id",
+        "position",
+        "measuring_quantity"
+    ],
+    "properties": {
+        "sensor_id": {
+            "description": "A unique identifier for the sensor.",
+            "type": "string"
+        },
+        "package_id": {
+            "description": "A unique identifier for the Waste Package where the sensor is placed.",
+            "type": "string"
+        },
+        "position": {
+            "description": "The position of the sensor defined by location and optional orientation.",
+            "type": "object",
+            "required": [
+                "location"
+            ],
+            "properties": {
+                "location": {
+                    "description": "The XYZ coordinates of the sensor.",
+                    "type": "array",
+                    "items": {
+                        "type": "number"
+                    }
+                },
+                "orientation": {
+                    "description": "The XYZ coordinates of the orientation.",
+                    "type": "array",
+                    "items": {
+                        "type": "number"
+                    }
+                }
+            }
+        },
+        "measuring_quantity": {
+            "description": "The type of measurement that the sensor is capable of taking.",
+            "type": "object",
+            "required": [
+                "type",
+                "unit"
+            ],
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "enum": [
+                        "temperature", "displacement", "radioactivity", "humidity"
+                    ]
+                },
+                "unit": {
+                    "description": "SI unit.",
+                    "type": "string"
+                }
+            }
+        },
+        "independent_variable": {
```

Comment:
I think this is very dependent on the model, for the experimental data, I would not make a difference between dependent and independent variables, but all are "measured" or deterministically prescribed

Reply:
Wanted to somehow keep information on the variable it is measured against. But this can go in the metadata of measured data